sandbox.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/docker/libnetwork/types"
 )
 
+const (
+	remoteCID = "remote"
```

Comment:
I was hoping it was not required to embed in the code the notion of remotely connected.

Given the purpose of this PR is to fix the `docker service` cli, which is not an critical issue being the cli in experimental and (mostly) knowing the network/service UI will go through changes, I would hold on on this PR.

It may even be this change will in future be required for a completely different reason.

Therefore I would keep it open until this is clarified.